actors/builtin/miner/miner_test.go

```diff
@@ -1194,14 +1194,18 @@ func TestProvingPeriodCron(t *testing.T) {
 	})
 
 	t.Run("detects and penalizes faults", func(t *testing.T) {
-		t.Skip("Disabled in miner state refactor #648, restore soon")
 		rt := builder.Build(t)
 		actor.constructAndVerify(rt)
 
-		allSectors := actor.commitAndProveSectors(rt, 2, 100, nil)
+		allSectors := actor.commitAndProveSectors(rt, 2, 181, nil)
+		pwr := miner.PowerForSectors(actor.sectorSize, allSectors)
+
+		// add lots of funds so penalties come from vesting funds
+		initialLocked := big.Mul(big.NewInt(200), big.NewInt(1e18))
```

Comment:
Does this make it easier to validate state?

Reply:
If we don't do this, the miner goes into IP debt and the the penalty we observe is just the amount of locked funds available. This just makes it easier to observe the correct penalty.